x-pack/platform/packages/shared/kbn-elastic-assistant-common/impl/schemas/anonymization_fields/bulk_crud_anonymization_fields_route.gen.ts

```diff
@@ -29,35 +32,89 @@ export type AnonymizationFieldsBulkActionSkipResult = z.infer<
   typeof AnonymizationFieldsBulkActionSkipResult
 >;
 export const AnonymizationFieldsBulkActionSkipResult = z.object({
+  /**
+   * id of the anonymization field that was not modified
+   */
   id: z.string(),
+  /**
+   * name of the anonymization field that was not modified
+   */
   name: z.string().optional(),
+  /**
+   * reason why the anonymization field was not modified
+   */
```

Comment:
Should these all start capitalized, or does it depend on if the first word is the field itself? E.g. `Reason`  would be capitalized, but not `name` in the field above this? Just curious for consistency sake here

Reply:
I'll fix that